cmd/gaiacli/README.md

```diff
@@ -0,0 +1,159 @@
+# IBC instruction
+
+// temporal document
+
+## Dependencies
+
+This branch uses non-canonical branch of cosmos-sdk. Before building, run `go mod vendor` on the root directory to retrive the dependencies. To build:
+
+```shell
+git clone git@github.com:cosmos/gaia
+cd gaia
+git checkout joon/ibc-gaia-interface
+go mod vendor
+make install
+gaiad version
+gaiacli version
+```
+
+Stub out testnet files for 2 nodes, this example does so in your $HOME directory:
+
+```shell
+cd ~ && mkdir ibc-testnets && cd ibc-testnet
```

Comment:
Wanted to point out that `cd ibc-testnet` is missing an "s"! Should be `cd ibc-testnets`